lircd.te

```diff
@@ -28,6 +28,8 @@ allow lircd_t self:process signal;
 allow lircd_t self:fifo_file rw_fifo_file_perms;
 allow lircd_t self:tcp_socket { accept listen };
 allow lircd_t self:netlink_kobject_uevent_socket create_socket_perms;
+allow lircd_t passwd_file_t:file { read getattr open };
+allow lircd_t self:capability { setuid setgid dac_override };
```

Comment:
I don't follow you here... Do you mean that  the last line isn't required, or should be rephrased (remember, this is selinux for dummies)